tests/questions/mc6.json

```diff
@@ -0,0 +1,23 @@
+[
+    {
+       "type": "Short",
```

Comment:
The convention is that the type of question is in the name of the test (this is not a `mc` multiple choice question). 

Please rename the test files to `shortFirstEmbedded` or something more precise.

Reply:
Will fix on my next push.